JS-08/ArrayMethods.js

```diff
@@ -0,0 +1,103 @@
+// 1. arr.at(index) is a newer feature that gives us the ability to use negative numbers to start from the end of the array (like in python).
+//    arr[index] for the most part does the same thing just doesnt know how to start from the end by using a negative index
+//    Example -> arr.at(-1) - This would get the last index in the array
+
+
+// Q2
+function createArrayFilledWith(letter, arrayLength) {
+    return new Array(arrayLength).fill(letter)
+}
+
+// Q3
+function removeElementsStartingFrom(n, arr) {
+    if (isNaN(n)) {
+        console.log("n must be a number")
+        return
+    } else if (n > arr.length) {
+        console.log("n cant be bigger then the array")
+        return;
+    }
+
+    return arr.slice(n)
+}
+
+// Q4
+function addToBeginningOfArray(value, arr) {
+    return arr.unshift(value)
```

Comment:
it seems like it doesn't return the modified array. Instead, it returns the new length of the array. To return the modified array, you should use arr.unshift(value); return arr;.